.github/workflows/e2e-tests.yaml

```diff
@@ -111,6 +121,27 @@ jobs:
       - name: Generate and Collect sosreport
         if: always()
         uses: ./.github/actions/sosreport-logs
+
+      # Generate snapshots (if enabled) {{{
+      - name: Retrieve product.txt from artifacts
+        if: inputs.generate-snapshots
+        run: >
+          curl --fail -LO -u ${{ secrets.ARTIFACTS_USER }}:${{ secrets.ARTIFACTS_PASSWORD }}
+          ${{ inputs.artifacts-url }}/product.txt
+      - name: Get full MetalK8s version
+        if: inputs.generate-snapshots
+        run: |
+          source product.txt
+          echo "METALK8S_VERSION=$VERSION" >> $GITHUB_ENV
+      - name: Generate snapshot
+        if: inputs.generate-snapshots
+        uses: ./.github/actions/generate-snapshots
+        with:
+          snapshot-name: "metalk8s-${{ env.METALK8S_VERSION }}-rocky-8-${{ env.NAME }}-simple environment"
```

Comment:
Do you remember why we introduced such complicated names? The suffix seems really redundant IMO (`simple environment` when we run `single-node`, and `3 nodes` when we run `multi-nodes`), wondering if we couldn't just give them a single "architecture" like in ARTESCA.

Reply:
The idea was to have a lot more snapshots in the future, but right I will keep it simple now that we are (kind of) refactoring it